package.json

```diff
@@ -51,7 +51,7 @@
     "eslint": "^7.9.0",
     "eslint-config-prettier": "^8.2.0",
     "eslint-plugin-jest": "^26.0.0",
-    "eslint-plugin-react": "^7.23.2",
+    "eslint-plugin-react": "^7.29.2",
```

Comment:
I had an issue with the `jsx-key` rule, it seems that there is an issue with this specific version and upgrading fix the issue